In file included from ../../gcc/gthr-posix.h:43,
from ./gthr-default.h:1,
from ../../gcc/gthr.h:114,
from ../../gcc/unwind-dw2.c:42:
/usr/include/pthread.h:664: error: array type has incomplete element type
make[3]: *** [libgcc/./unwind-dw2_s.o] Error 1
						   
--- glibc-2.3.4/nptl/sysdeps/pthread/pthread.h.orig	2004-11-24 23:55:24.000000000 +0100
+++ glibc-2.3.4/nptl/sysdeps/pthread/pthread.h	2005-02-14 16:24:22.104239520 +0100
@@ -24,9 +24,9 @@
 #include <time.h>
 
 #define __need_sigset_t
+#include <setjmp.h>
 #include <signal.h>
 #include <bits/pthreadtypes.h>
-#include <bits/setjmp.h>
 #include <bits/wordsize.h>
 
 
@@ -660,7 +660,6 @@
 #endif
 
 /* Function used in the macros.  */
-struct __jmp_buf_tag;
 extern int __sigsetjmp (struct __jmp_buf_tag __env[1], int __savemask) __THROW;
 
 
